provider/resource_service_test.go

```diff
@@ -607,6 +607,6 @@ func assertServiceResourceJSONEq(t *testing.T, expected string, actual string, m
 			return kpis[i].(map[string]interface{})["_key"].(string) > kpis[j].(map[string]interface{})["_key"].(string)
 		})
 	}
-
+	delete(actualJSONAsInterface, "_key")
```

Comment:
I think it's better to add _key into actualJSONAsInterface. Pass some _key as additional param
and add this to the base:
```
// testOverridable
var GenerateUUID = func() (string, error) {
	return uuid.GenerateUUID()
}
```
and then override it in the test method